From 0210f585ad158b34fc2aaae53e55318de58a5f92 Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Thu, 14 Mar 2019 13:41:19 -0700
Subject: [PATCH 09/31] UBUNTU: SAUCE: Revert "apparmor: fix checkpatch error
 in Parse secmark policy"

Revert currently unused by Ubuntu apparmor features that require secid
support, to enable apparmor LSM stacking with the 5.1 LSM stacking
patchset.

This reverts commit 76af016e65e74023a11b2710e18a98b68b86e3cd.

Signed-off-by: John Johansen <john.johansen@canonical.com>
Signed-off-by: Seth Forshee <seth.forshee@canonical.com>
---
 security/apparmor/policy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/apparmor/policy.c b/security/apparmor/policy.c
index 58ff956aa6b3..722e6d516d03 100644
--- a/security/apparmor/policy.c
+++ b/security/apparmor/policy.c
@@ -227,7 +227,7 @@ void aa_free_profile(struct aa_profile *profile)
 	for (i = 0; i < profile->xattr_count; i++)
 		kzfree(profile->xattrs[i]);
 	kzfree(profile->xattrs);
-	for (i = 0; i < profile->secmark_count; i++)
+	for (i=0; i < profile->secmark_count; i++)
 		kzfree(profile->secmark[i].label);
 	kzfree(profile->secmark);
 	kzfree(profile->dirname);
-- 
2.25.1

